lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb

```diff
@@ -541,6 +541,13 @@ def self.convert_value_based_on_datatype(value, datatype)
       value
     end
 
+    def self.fetch_password(value)
```

Comment:
CodeClimate reports: "`private` (on line 430) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead."

Is this intended to be a private method?